routes/web.php

```diff
@@ -77,18 +77,18 @@
 
 //guest info
 Route::get('/guestinfo/guest-form', function () {
-    return view('guest-information/Guest Information Form');
+    return view('guest-information.guest_information_form');
 });
 
 Route::get('/guestinfo/reservation', function () {
-    return view('guest-information/Reservation');
+    return view('guest-information.reservation');
```

Comment:
walang 'reservation' na blade template, 'reservation_form' ata to

Reply:
fixed